tests/parser/test_csv_parser.py

```diff
@@ -0,0 +1,10 @@
+from sampo.parser.csv_parser import CSVParser
+from sampo.parser.exception import WorkGraphBuildingException
+
+
+def test_work_graph_csv_parser():
+    try:
+        wg = CSVParser.from_csv('parser/test_work_graph.csv')
+        print(f'\n\nWork graph has works: {wg.nodes}\n\n')
+    except Exception as e:
+        raise WorkGraphBuildingException('There is no way to build work graph')
```

Comment:
Test file should have name that ends with `test` to be collected with `pytest` runner